src/components/pages/navbar/NavbarContainer.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Layout, Menu } from 'antd';
+import {
+  HomeOutlined,
+  LogoutOutlined,
+  SettingFilled,
+  FormOutlined,
+  EditOutlined,
+} from '@ant-design/icons';
+import {
+  EDIT_PROGRAM_PATH,
+  SETTINGS_PATH,
+  // CREATE_COURSE_PATH,
+  CREATE_COURSE_PAGE_PATH,
+} from '../../../routes';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: inherit;
+`;
+
+// Wrap `Menu.Item` from AntD in a `Link` from react-router-dom
+// We can just toss in a "to" attribute to MenuItem
+// and it will look like an AntD Menu.Item but act like a Link
+const MenuItem = ({ key, to, icon, children, ...props }) => {
```

Comment:
I really like what you've done here with this pattern. This is a great way to add the appropriate kind of nav item if it has a path or not while incorporating Routes from React Router. Very elegantly done!